integr8ly/cloud-resource-operator
# Slow reconcile on success

## Verification

 - Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Ensure a blob storage reconcile completes successfully
